base/strings/util.jl

```diff
@@ -711,3 +711,12 @@ julia> ascii("abcdefgh")
 ```
 """
 ascii(x::AbstractString) = ascii(String(x))
+
+rest(s::Union{String,SubString{String}}, i=1) = SubString(s, i)
+function rest(s::AbstractString, st...)
+    io = IOBuffer(; maxsize=ncodeunits(s))
```

Comment:
Is specifying `maxsize` to always be `ncodeunits(s)` here a good idea? I think it should always be correct, but I am not sure whether it's actually efficient for different string encodings.

Reply:
Oh, I think this is wrong for types like [`UTF16String`](https://github.com/JuliaStrings/LegacyStrings.jl/blob/8d9fd731767d4c86c7b71210b259d45045840d9c/src/utf16.jl#L52), so let's remove this for now.